### PR TITLE
Jax-RS Jersey3 Server

### PR DESCRIPTION
**Pull request by CDerksen**
Hi There!
As the name indicates, these are adjustments to generate server stubs for JAX-RS Jersey3 and OpenApi 3.x.

Esspecially, with the changes to OpenApi 3.x I would be glad, if somebody could have a closer look to the annotations produced, or to name some further test cases to be used for the last improvments.
For example: The api template (see ./modules/openapi-generator/src/main/resources/JavaJaxRS/libraries/jersey3/api.mustache) still contains #implicitHeadersParams (ApiImplicitParams) from Swagger1. They were not used during my tests and I was not sure what to do with them. 

Furthermore, this PR is related to the branch and PR 'jaxrs-jersey3' from May 2022 that was not improved anymore and (most likely) might be closed. [see jaxrs-jersey3](https://github.com/OpenAPITools/openapi-generator/tree/jaxrs-jersey3)

### PR checklist
 
- [x ] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x ] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
**Actually I didn't added a further example yet, since the code produced seems to work.**
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh
  ./bin/utils/export_docs_generators.sh
  ``` 
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] In case you are adding a new generator, run the following additional script : 
  ```
  ./bin/utils/ensure-up-to-date
  ``` 
  Commit all changed files.
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master` (6.3.0) (minor release - breaking changes with fallbacks), `7.0.x` (breaking changes without fallbacks)
- [x] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.
@bbdouglas (2017/07) @sreeshas (2017/08) @jfiala (2017/08) @lukoyanov (2017/09) @cbornet (2017/09) @jeff9finger (2018/01) @karismann (2019/03) @Zomzog (2019/04) @lwlee2608 (2019/10) @martin-mfg (2023/08)

### CODE REVIEW

**Comment by CDerksen** [comment]
Dear All,
I think I could quickly solve the tab issue, but with the 
`assert petApiService.text.contains("import javax.ws.rs.core.Response;")` 
I'm not sure. Seems that a test case or a case separation is misssing!?

Any suggestions? How to proceed?

Beste wishes,
Christian

File: .github/.test/samples.json [hunk 1]
```diff
@@ -800,12 +800,6 @@
         "Client: Python"
       ]
     },
-    {
-      "input": "python-server-flask-petstore-python2.sh",
-      "matches": [
-        "Server: Python"
-      ]
-    },
```
Comment (wing328) [inline .github/.test/samples.json:808]: looks like a rebase/merge was done incorrectly to include these changes that are already present in the master

Comment (CDerksen) [reply .github/.test/samples.json:808]: Yes I used a rebase/merge ... bad!
What's the best way to solve this issue?

Comment (wing328) [reply .github/.test/samples.json:808]: one way is to create a new PR and cherry-pick commits.

Comment (CDerksen) [reply .github/.test/samples.json:808]: Done! - Please have a look to [the new PR for jaxrs jersey3-server](https://github.com/OpenAPITools/openapi-generator/pull/16335)

**Pull request closed without merging**